### PR TITLE
Convert to Swift 3

### PR DESCRIPTION
**Pull request by ANNotunzdY**

### CODE REVIEW

**Comment by joeboyscout04** [comment]
+1 would be great to add this since Xcode 8 is released.

**Comment by dbettermann** [comment]
I'm thinking the Swift 3 upgrade should be on a separate branch and Swift 2.3 should be merged into master. That's the way I've seen a lot of other repos do it.

@ANNotunzdY you ok with changing the base branch?

**Comment by ANNotunzdY** [comment]
@dbettermann OK, I'll change the base. Please create a branch for swift 3.

**Comment by dbettermann** [comment]
It is created.

**Review by ataranlen — APPROVED** [review]
Tested on XCode 8, compiling against the current swift language version.

File: Source/DBAlertController.swift [hunk 1]
```diff
@@ -38,12 +38,12 @@ public class DBAlertController: UIAlertController {
...
     
-    private override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return UIApplication.sharedApplication().statusBarStyle
+    fileprivate override var preferredStatusBarStyle : UIStatusBarStyle {
+        return UIApplication.shared.statusBarStyle
     }
     
-    private override func prefersStatusBarHidden() -> Bool {
-        return UIApplication.sharedApplication().statusBarHidden
+    fileprivate override var prefersStatusBarHidden : Bool {
```
Comment (dbettermann) [inline Source/DBAlertController.swift]: Can you remove the space before the colon? I try to follow the Ray Wenderlich style guide. (https://github.com/raywenderlich/swift-style-guide#spacing)

File: Source/DBAlertController.swift [hunk 2]
```diff
@@ -38,12 +38,12 @@ public class DBAlertController: UIAlertController {
 // In the case of view controller-based status bar style, make sure we use the same style for our view controller
 private class DBClearViewController: UIViewController {
     
-    private override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return UIApplication.sharedApplication().statusBarStyle
+    fileprivate override var preferredStatusBarStyle : UIStatusBarStyle {
```
Comment (dbettermann) [inline Source/DBAlertController.swift]: Can you remove the space before the colon? I try to follow the Ray Wenderlich style guide. (https://github.com/raywenderlich/swift-style-guide#spacing)

**Review by dbettermann — CHANGES_REQUESTED** [review]
Make the two small changes I commented about and this branch can be merged in.

**Comment by dbettermann** [comment]
I updated the master branch to support Swift 2.3, so it looks like you'll have to fix a few small conflicts.

File: Source/DBAlertController.swift (hunk 2 above)
Comment (ANNotunzdY) [reply Source/DBAlertController.swift]: Sorry, I fixed it.

**Review by dbettermann — APPROVED** [review]

**Comment by dbettermann** [comment]
Once you resolve your merge conflicts I can merge it in.

**Comment by ANNotunzdY** [comment]
I merged the conflicts.

**Comment by dbettermann** [comment]
Thanks!

**Pull request merged**